content/en/database_monitoring/setup_oracle/_index.md

```diff
@@ -24,26 +24,93 @@ The features described on this page are in private beta.
 The following deployment configurations, components, and features are not supported: Oracle Autonomous Database, ASM, and Data Guard.
 
 
-## Supported Agent versions
+## Latest Agent version
 
-The following custom build is supported: `7.44.1~dbm~oracle~beta~0.28`.
+The following beta build contains implemented Oracle DBM features: `7.44.1~dbm~oracle~beta~0.28`.
```

Comment:
Can we add a link here? i.e. to the [custom build repo](https://hub.docker.com/r/datadog/agent/tags?page=1&name=oracle)

Also, should a `-` be used instead of the `~` symbol? Just going off of what's in the repo linked above

Reply:
> Can we add a link here?
There is a repo for each OS. Should I rather put the references in the [section which explains how to install the beta build](https://github.com/DataDog/documentation/pull/18688/files#diff-cc20dbc1da7eeca1fc6a976fb830286772bd774cd607420503f16b40ef462e22R71-R101) or here?

> Also, should -s be used instead of the ~ symbol
Unfortunately, it isn't unified. `-` is used only for Docker images, and `~` everywhere else.

Reply:
Would it be redundant to include it in both places? If you think it would be, then the section that explains how to install the beta build is a fine place for them

Reply:
Actually, I think the inclusion of this whole section ("Latest Agent version") may be redundant given the "Install Agent" header under "Prerequisites"

Reply:
Here's my proposal: move this section to be below the "Docker" box and before the "Oracle client" section (it's a bit awkward to have the link take you up in the page rather than down)

Reply:
And then I think you can put the references in the section you identified ("section which explains how to install the beta build")

Reply:
I made the change.